### PR TITLE
WT-8915 Change log verbosity level mapping between AWS SDK and WiredTiger

### PR DESCRIPTION
**Pull request by Sean04**
This change fixes the python test failures due to aws calls intermittently timing out by changing the aws SDK log levels, such that only fatal errors are output to stderr by default. The aws log levels are now mapped to the WiredTiger verbosity levels as follows:
AWS::Fatal -> WT_VERBOSE_ERROR
AWS::Error -> WT_VERBOSE_WARNING
AWS::Warn -> WT_VERBOSE_WARNING
AWS::Info -> WT_VERBOSE_INFO
AWS::Debug -> WT_VERBOSE_DEBUG

### CODE REVIEW

File: test/suite/test_tiered06.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def conn_extensions(self, extlist):
         # S3 store is built as an optional loadable extension, not all test environments build S3.
         if self.ss_name == 's3_store':
             #config = '=(config=\"(verbose=1)\")'
+            # config = '=(config=\"(verbose=[api:1,version,tiered:-3])\")'
```
Comment (sulabhM) [inline test/suite/test_tiered06.py:62]: Could you keep one of the commented out "config" and remove the other ones. Maybe merge them together and say ```tiered:1``` ?

File: test/suite/test_tiered14.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def conn_extensions(self, extlist):
         config = ''
         # S3 store is built as an optional loadable extension, not all test environments build S3.
         if self.ss_name == 's3_store':
-            #config = '=(config=\"(verbose=1)\")'
+            # config = '=(config=\"(verbose=[api:1,version,tiered:-3])\")'
```
Comment (sulabhM) [inline test/suite/test_tiered14.py:89]: Same comment as above, please only keep one of the commented "config"

**Review by sulabhM — APPROVED** [review]
I have a minor comment, pending that this change LGTM!

File: test/suite/test_tiered06.py [hunk 3]
```diff
@@ -105,8 +106,7 @@ def test_ss_basic(self):
 
         # The object doesn't exist yet.
         if self.ss_name == 's3_store':
-            with self.expectedStderrPattern('.*HTTP response code: 404.*'):
-                self.assertFalse(fs.fs_exist(session, 'foobar'))
+            self.assertFalse(fs.fs_exist(session, 'foobar'))
```
Comment (tammybailey) [inline test/suite/test_tiered06.py:109]: We don't need the if-else here. We can remove lines 108, 110 and 111.

File: test/suite/test_tiered06.py [hunk 4]
```diff
@@ -116,8 +116,7 @@ def test_ss_basic(self):
 
         # The object still doesn't exist yet.
         if self.ss_name == 's3_store':
```
Comment (tammybailey) [inline test/suite/test_tiered06.py:118]: Same as above

**Review by tammybailey — APPROVED** [review]
Two small cleanups requested, else LGTM!

File: test/suite/test_tiered06.py (hunk 1 above)
Comment (Sean04) [reply test/suite/test_tiered06.py:62]: I've just changed it to keep `config = '=(config=\"(verbose=[api:1,version,tiered:1])\")'`

File: test/suite/test_tiered14.py (hunk 2 above)
Comment (Sean04) [reply test/suite/test_tiered14.py:89]: This one already removed the first `config`

File: test/suite/test_tiered06.py (hunk 3 above)
Comment (Sean04) [reply test/suite/test_tiered06.py:109]: Fixed.

File: test/suite/test_tiered06.py (hunk 4 above)
Comment (Sean04) [reply test/suite/test_tiered06.py:118]: Fixed.

**Pull request merged**